[2023-03-12T22:42:37.406+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: validacao_pu.task_calcular_pu_externo_xyz scheduled__2023-03-05T00:00:00+00:00 [queued]>
[2023-03-12T22:42:37.452+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: validacao_pu.task_calcular_pu_externo_xyz scheduled__2023-03-05T00:00:00+00:00 [queued]>
[2023-03-12T22:42:37.453+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-12T22:42:37.453+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-03-12T22:42:37.453+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-12T22:42:37.562+0000] {taskinstance.py:1300} INFO - Executing <Task(_PythonDecoratedOperator): task_calcular_pu_externo_xyz> on 2023-03-05 00:00:00+00:00
[2023-03-12T22:42:37.571+0000] {standard_task_runner.py:55} INFO - Started process 1526 to run task
[2023-03-12T22:42:37.578+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'validacao_pu', 'task_calcular_pu_externo_xyz', 'scheduled__2023-03-05T00:00:00+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/examples/validacao_pu.py', '--cfg-path', '/tmp/tmpcahjrzwb']
[2023-03-12T22:42:37.579+0000] {standard_task_runner.py:83} INFO - Job 124: Subtask task_calcular_pu_externo_xyz
[2023-03-12T22:42:37.985+0000] {task_command.py:388} INFO - Running <TaskInstance: validacao_pu.task_calcular_pu_externo_xyz scheduled__2023-03-05T00:00:00+00:00 [running]> on host 40565fc59ef6
[2023-03-12T22:42:38.473+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alexandre.yoshimatsu
AIRFLOW_CTX_DAG_ID=validacao_pu
AIRFLOW_CTX_TASK_ID=task_calcular_pu_externo_xyz
AIRFLOW_CTX_EXECUTION_DATE=2023-03-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-05T00:00:00+00:00
[2023-03-12T22:42:38.476+0000] {logging_mixin.py:137} INFO - Operação: xyz
[2023-03-12T22:42:38.477+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-12T22:42:38.504+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=validacao_pu, task_id=task_calcular_pu_externo_xyz, execution_date=20230305T000000, start_date=20230312T224237, end_date=20230312T224238
[2023-03-12T22:42:38.959+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-12T22:42:39.108+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
